src/routes/boot.ts

```diff
@@ -346,6 +350,9 @@ const getAnonymousFirstVisit = async (trackingId: string) => {
   return finalValue;
 };
 
+// We released the firstVisit at July 10, 2023 - so to have a buffer time, the requirement is the day after.
+export const onboardingV2Requirement = new Date(2023, 6, 11);
```

Comment:
We will have to update it once we want to start the experiment. Just noting it here to remember

Reply:
Could we not created flagsmith flag for these kind of things? Feels clunky to leave it like this.